README.md

```diff
@@ -307,6 +328,18 @@ public interface ExampleHandler {
 }
 ```
 
+You also can make your handler extended from some other handler class by
+setting base class as an annotation parameter. For example, if you want the
+messages to be handled on separate thread, you can set
+`ExecutableHandler.class` as a `@Handler` parameter:
```

Comment:
Lets add explicitly that the generated class will respect ExecutableHandler interface and generate its code in run() method instead of handle method.